src/bridge/SequencerInbox.sol

```diff
@@ -99,6 +101,13 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         _;
     }
 
+    modifier onlyRollupOwnerOrBatchPosterManager() {
+        if (msg.sender != rollup.owner() && msg.sender != batchPosterManager) {
+            revert NotBatchPosterManager(msg.sender);
```

Comment:
maybe `NotBatchPosterManagerOrRollupOwner`? (maybe fine as is)

Reply:
ikr but intentionally used the short form